build.gradle

```diff
@@ -324,8 +291,6 @@ bundle.doLast() {
 
 tasks.assemble.dependsOn(bundle)
 tasks.bundle.dependsOn(createPluginDescriptor)
-tasks.assemble.dependsOn(bundleSecurityAdminStandalone)
```

Comment:
> So this one does not present a maintenance overhead (as it is just an additional gradle action)
@davidlago 

Due to these lines, they are built by default.  Would you prefer I leave the tasks, but remove these lines so you have to opt in with `./gradlew bundleSecurityAdminStandaloneTarGz`.  Note; this feature has no documentation in this codebase which is why I figured it would make more sense to remove it entirely.

Reply:
I think that's a good compromise. Let's move to non-default first to put the potential users "on notice". If they try to run it and it is not built by default they have an easy fix, and we can think of removing in 2.0. Do you have any ideas for how to do the deprecation notice @peternied ?

Reply:
Note: this should give us an opportunity to engage with people using this, if anyone, to make sure we understand the use case and perhaps have an argument against deprecation/removal.

Reply:
> Do you have any ideas for how to do the deprecation notice?

Typically for a breaking change we have the weekly meeting that is a great place to communicate them.  However, for this change it is murkier since we hardly have supported this tool.  I've created a [post in the OpenSearch forums](https://discuss.opendistrocommunity.dev/t/security-admin-standalone-should-there-be-more-investment-in-this-tool/8748) as a starting point.  @stockholmux Did you have additional thoughts on how we might want to spread the word or gather input?

Reply:
I've seen no traction over the past few days, unless we have other thoughts I'd like to disable building by default and we can add it back if there is concern.